lib/livebook_web/live/hub/edit/personal_component.ex

```diff
@@ -0,0 +1,82 @@
+defmodule LivebookWeb.Hub.Edit.PersonalComponent do
+  use LivebookWeb, :live_component
+
+  alias Livebook.Hubs.Personal
+
+  @impl true
+  def update(assigns, socket) do
+    changeset = Personal.change_hub(assigns.hub)
+
+    {:ok,
+     socket
+     |> assign(assigns)
+     |> assign(changeset: changeset)}
+  end
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div id={"#{@id}-component"}>
+      <div class="flex flex-col space-y-10">
+        <div class="flex flex-col space-y-2">
+          <h2 class="text-xl text-gray-800 font-medium pb-2 border-b border-gray-200">
+            General
+          </h2>
+
+          <.form
+            :let={f}
+            id={@id}
+            class="flex flex-col mt-4 space-y-4"
+            for={@changeset}
+            phx-submit="save"
+            phx-change="validate"
+            phx-target={@myself}
+            phx-debounce="blur"
+          >
+            <div class="grid grid-cols-1 md:grid-cols-2 gap-3">
+              <.input_wrapper form={f} field={:hub_name} class="flex flex-col space-y-1">
+                <div class="input-label">Name</div>
+                <%= text_input(f, :hub_name, class: "input") %>
+              </.input_wrapper>
+
+              <.input_wrapper form={f} field={:hub_emoji} class="flex flex-col space-y-1">
+                <div class="input-label">Emoji</div>
+                <.emoji_input
+                  id="fly-emoji-input"
```

Comment:
```suggestion
                  id="personal-emoji-input"
```

or perhaps just change to `emoji-input` in both to ease copy and paste. :)